third-party/eazy/v1/openapi.yaml

```diff
@@ -323,21 +323,21 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/responses/apiKeysResponse'
+                  $ref: '#/components/schemas/apiKeysResponse'
```

Comment:
Is this correct?

Reply:
`#/components/schemas/apiKeyResponse` is correct

Reply:
Yeah, the `apiKeysResponse` part is fixed in another commit.